llvm/unittests/ProfileData/InstrProfTest.cpp

```diff
@@ -370,12 +370,53 @@ static CallStackIdMapTy getCallStackMapping() {
   return Mapping;
 }
 
+// Populate all of the fields of MIB.
+MemInfoBlock makeFullMIB() {
```

Comment:
I think we should use the macro expansion of fields here so that it's future proof i.e. any new fields added to the original MIBEntry will automatically get picked up.